### DISCUSSION TITLE
Set permanent word wrap option

### DISCUSSION THREAD

**Discussion by himat** [dbeaver/dbeaver#10996] [Ideas]
In the inline data viewer, I can manually set the word wrap option to wrap lines, but this setting is lost when I close and re-open dbeaver.

![image](https://user-images.githubusercontent.com/1156974/104796797-6fa1ac80-5787-11eb-9ac5-a3edb7dc3fb5.png)

Is there a preference setting to make lines wrap by default?

**Comment by dp250195** [comment]
Yeah, this is pretty annoying, having to re-enable word-wrap every time the query is ran. We'd really appreciate if an option to have this always enabled is added to application preferences.

**Comment by E1izabeth** [comment]
You could create a feature request issue. I suppose it's a good idea to save the state of this option.

**Comment by E1izabeth** [comment]
I've found an existing issue that cause the resetting of Word wrap option https://github.com/dbeaver/dbeaver/issues/19344.
